rasa/core/actions/forms.py

```diff
@@ -615,12 +615,14 @@ async def _validate_if_required(
             tracker.latest_action_name == ACTION_LISTEN_NAME
             and not tracker.active_loop.get(LOOP_INTERRUPTED, False)
         )
+
         if needs_validation:
             logger.debug(f"Validating user input '{tracker.latest_message}'.")
             return await self.validate(tracker, domain, output_channel, nlg)
-
-        logger.debug("Skipping validation.")
-        return []
+        else:
+            # Needed to determine which slots to request although there are no slots to actually validate
```

Comment:
could you please wrap the line so that it not exceeds 88 chars? That's the limit for our code and I think we should also it for the comments (weird that this isn't enforced 🤔 )

Reply:
Sure, done!